jwst/ramp_fitting/ramp_fit.py

```diff
@@ -1756,14 +1796,21 @@ def calc_opt_fit(nreads_wtd, sumxx, sumx, sumxy, sumy):
     sig_intercept = (sumxx / denominator)**0.5
     sig_slope = (nreads_wtd / denominator)**0.5
 
+    # variance of slope due to Poisson noise only, and
+    # variance of slope due to Read noise only
+    sig_slope_p = slope/(mask_2d.sum(axis=0) - 1.)
+    sig_slope_r = nreads_wtd/denominator 
```

Comment:
`nreads_wtd` is the sum of the Fixsen weights (I think also divided by the read noise).  In this case, the variance computed from just read noise should be `(ngroups / readnoise^2) / denominator`.